html/Mogo/js/accordion/accordion.js

```diff
@@ -0,0 +1,36 @@
+import {components} from './components.js';
+import {hideEveryPanel, hidePanel, showPanel} from './helper-functions.js';
+
+export function addAccordionEffect() {
+  components.then((response) => {
+    const {accordion, panelElements, imgElement} = response;
+
+    let visibleAccordionId = null;
+    let currentClickedAccorionId = null;
+
+    for (let i = 0; i < accordion.length; i++) {
+      accordion[i].addEventListener('click', (event) => {
+        currentClickedAccorionId = event.target.id;
+        hideEveryPanel(panelElements, accordion);
+
+        if (currentClickedAccorionId !== visibleAccordionId) {
+          event.target.classList.toggle('accordion--active');
+          event.target.children[1].classList.toggle('accordion__arrow--active');
+          imgElement.src = './assests/images/services/img-' + i + '.jpg';
```

Comment:
generally it would be better to have array with links and use it as it can be more flexible and you will not have to follow pattern with image naming(and explain it for everyone in your team/or content team)